### PR TITLE
Expose all template parameters during mlt build

### PR DESCRIPTION
**Pull request by karthikvadla**
fixes #{issue_num_here_if_applicable}

If you aren't done, please add the `wip` label until you're ready for review.

If you are adding a new template or mlt command/flag ensure that is is documented in the [docs/features.md](https://github.com/IntelAI/mlt/tree/master/docs/features.md) file.

## Reviewer Checklist

 - [ ] Do you understand it?
 - [ ] Is it correct?
 - [ ] Is it safe?
 - [ ] Is it legally compliant?
 - [ ] Is it robust?
 - [ ] Is it simple?
 - [ ] Is it tested?
 - [ ] Is it documented?
 - [ ] Is the style consistent?

See [the reviewer guide](docs/reviews.md) for more detail on each check.

### CODE REVIEW

File: mlt/commands/build.py [hunk 1]
```diff
@@ -52,8 +52,13 @@ def _build(self):
...
         template_parameters = config_helpers.\
             get_template_parameters(self.config)
-        build_cmd = "CONTAINER_NAME={} GPUS={} make build".format(
-            container_name, template_parameters.get('gpus', 0))
+
+        params = ""
+        for key, val in template_parameters.items():
+            params += "{}={} ".format(key.upper(), val)
+
+        build_cmd = "CONTAINER_NAME={} {}make build".format(
```
Comment (s1113950) [inline mlt/commands/build.py]: If there is a case where there are no template params this is gonna print weirdly

Comment (dmsuehir) [reply mlt/commands/build.py]: Wouldn't `params = ""` if there are no template params?  If so, it wouldn't be weird, right?

Comment (karthikvadla) [reply mlt/commands/build.py]: yeah that makesense

Comment (s1113950) [reply mlt/commands/build.py]:
```
In [130]: params = ""

In [131]: template_parameters = {}

In [132]: for key, val in template_parameters.items():
     ...:                 params += "{}={} ".format(key.upper(), val)
     ...:

In [133]: build_cmd = "container_name={} {}make build".format("name", params)

In [134]: build_cmd
Out[134]: 'container_name=name make build'
``` 

🤦‍♂️ no @karthikvadla you're good here, I didn't see that you were converting it to a string. I thought you were printing it out 😅 in this case you don't have random space when using `format`:

```
In [137]: va = "something {}frogs".format("")
In [138]: va
Out[138]: 'something frogs'
```

so nvmd :homer_disappear:

File: tests/unit/commands/test_build.py [hunk 2]
```diff
@@ -138,3 +143,55 @@ def test_build_verbose(popen_mock, open_mock, init_mock):
     built = output.find('Built')
     assert all(var >= 0 for var in (starting, built))
     assert starting < built
+
+
+def test_build_get_template_parameters_empty(progress_bar_mock,
```
Comment (s1113950) [inline tests/unit/commands/test_build.py]: These two are almost exactly the same except for `get_template_parameters_mock` it looks like. Can you parametrize these tests together?

File: mlt/commands/build.py [hunk 3]
```diff
@@ -52,8 +52,14 @@ def _build(self):
 
         template_parameters = config_helpers.\
             get_template_parameters(self.config)
-        build_cmd = "CONTAINER_NAME={} GPUS={} make build".format(
-            container_name, template_parameters.get('gpus', 0))
+
+        params = ""
+        if template_parameters:
```
Comment (s1113950) [inline mlt/commands/build.py]: No need for this `if` anymore, see thread below

**Pull request merged**